src/components/MainPage.js

```diff
@@ -1,3 +1,5 @@
+// Cleaned up the bottom to remove excess code.  Added comments for methods  -Radium Branch
```

Comment:
- Cleaned up the bottom to remove excess code.  
- Added comments for methods to explain what they do.